toc.md

```diff
@@ -0,0 +1,11 @@
+### Table Of Contents
```

Comment:
ok - this means I also have to change my script to add a new lesson. But having a separate toc.md looks a lot better than always changing lesson1.md :) I just wish there was a plugin that could auto-generate the toc based on `#` headers...

Reply:
I updated the `newLesson.js` script...
honestly if I had time I would have written the plugin, it doesn't seem so complicated.